content/frontend/react-apollo/8-subscriptions.md

```diff
@@ -1,201 +1,225 @@
+means when working with subscriptions, we're breaking the
+_Request-Response_ cycle that is typically used for
+interactions with the API. Instead, the client now initiates
+a steady connection with the server by specifying which
+event it is interested in. Every time this particular event
+then happens, the server uses the connection to push the
+expected data to the client.
 
 ### Subscriptions with Apollo
 
-When using Apollo, you need to configure your `ApolloClient` with information about the subscriptions endpoint. This is done by adding another `ApolloLink` to the Apollo middleware chain. This time, it's the `WebSocketLink` from the [`apollo-link-ws`](https://github.com/apollographql/apollo-link/tree/master/packages/apollo-link-ws) package.
+When using Apollo, we need to configure our `ApolloClient`
+with information about the subscriptions endpoint. This is
+done by adding another `ApolloLink` to the Apollo middleware
+chain. This time, it's the `WebSocketLink` from the
+[`@apollo/client/link/ws`](https://github.com/apollographql/apollo-link/tree/master/packages/apollo-link-ws)
+package.
 
-Go and add this dependency to your app first.
+To get started, add `subscriptions-transport-ws` as a
+dependency to the app.
 
 <Instruction>
 
-Open a terminal and navigate to the project's root directory. Then execute the following command:
+Open a terminal and navigate to the project's root
+directory. Then execute the following command:
 
 ```bash(path=".../hackernews-react-apollo")
-yarn add apollo-link-ws
-```
-
-> **Note**: For apollo-link-ws to work you also need to install subscriptions-transport-ws
-
-```
 yarn add subscriptions-transport-ws
 ```
 
 </Instruction>
 
-Next, make sure your `ApolloClient` instance knows about the subscription server.
+Next, let's make sure our `ApolloClient` instance knows
+about the subscription server.
 
 <Instruction>
 
-Open `index.js` and add the following import to the top of the file:
+Open `index.js` and add the following imports to the top of
+the file:
 
 ```js(path=".../hackernews-react-apollo/src/index.js")
-import { split } from 'apollo-link'
-import { WebSocketLink } from 'apollo-link-ws'
-import { getMainDefinition } from 'apollo-utilities'
+import { split } from '@apollo/client';
+import { WebSocketLink } from '@apollo/client/link/ws';
+import { getMainDefinition } from '@apollo/client/utilities';
 ```
 
-</Instruction>
-
-Notice that you're now also importing the `split` function from 'apollo-link'.
-
-<Instruction>
-
-Now create a new `WebSocketLink` that represents the WebSocket connection. Use `split` for proper "routing" of the requests and update the constructor call of `ApolloClient` like so:
+Let's now create a new `WebSocketLink` that represents the
+WebSocket connection. We'll use `split` for proper "routing"
+of the requests and update the constructor call of
+`ApolloClient` like so:
 
 ```js(path=".../hackernews-react-apollo/src/index.js")
```

Comment:
I think it would be helpful to include a diff in this section. I got lost for a moment.